[2022-06-16 10:32:08,564] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-06-16 10:32:08,661] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-06-16 10:32:08,677] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:32:08,678] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-16 10:32:08,679] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:32:08,816] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): getdata> on 2022-05-11 00:00:00+00:00
[2022-06-16 10:32:08,850] {standard_task_runner.py:52} INFO - Started process 3550 to run task
[2022-06-16 10:32:08,952] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'analytics', 'getdata', 'scheduled__2022-05-11T00:00:00+00:00', '--job-id', '431', '--raw', '--subdir', 'DAGS_FOLDER/testing_etl.py', '--cfg-path', '/tmp/tmp9ze2me1d', '--error-file', '/tmp/tmp1ol8xwue']
[2022-06-16 10:32:08,972] {standard_task_runner.py:80} INFO - Job 431: Subtask getdata
[2022-06-16 10:32:09,662] {task_command.py:370} INFO - Running <TaskInstance: analytics.getdata scheduled__2022-05-11T00:00:00+00:00 [running]> on host fe209149cfc1
[2022-06-16 10:32:10,425] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=analytics
AIRFLOW_CTX_TASK_ID=getdata
AIRFLOW_CTX_EXECUTION_DATE=2022-05-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-11T00:00:00+00:00
[2022-06-16 10:32:10,429] {mysql.py:82} INFO - Executing: SELECT * from Customer_T;
[2022-06-16 10:32:10,570] {base.py:68} INFO - Using connection ID 'mysql_con' for task execution.
[2022-06-16 10:32:10,719] {dbapi.py:213} INFO - Running statement: SELECT * from Customer_T;, parameters: None
[2022-06-16 10:32:10,731] {dbapi.py:221} INFO - Rows affected: 19
[2022-06-16 10:32:10,951] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=analytics, task_id=getdata, execution_date=20220511T000000, start_date=20220616T103208, end_date=20220616T103210
[2022-06-16 10:32:11,237] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 10:32:11,561] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
